tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -51,56 +51,63 @@ static void to_display_format(const u8_t *src, size_t size, char *dst)
 	}
 }
 
-struct spi_config spi_slow = {
+struct spi_config spi_cfg_slow = {
 	.frequency = SLOW_FREQ,
 	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
 	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
 	.slave = SPI_SLAVE,
 	.cs = SPI_CS,
 };
 
-struct spi_config spi_fast = {
+struct spi_config spi_cfg_fast = {
 	.frequency = FAST_FREQ,
 	.operation = SPI_OP_MODE_MASTER | SPI_MODE_CPOL |
 	SPI_MODE_CPHA | SPI_WORD_SET(8) | SPI_LINES_SINGLE,
 	.slave = SPI_SLAVE,
 	.cs = SPI_CS,
 };
 
-static int cs_ctrl_gpio_config(struct spi_cs_control *cs)
+#if defined(CONFIG_SPI_LOOPBACK_CS_GPIO)
+static int cs_ctrl_gpio_config(void)
 {
-	if (cs) {
-		cs->gpio_dev = device_get_binding(CS_CTRL_GPIO_DRV_NAME);
-		if (!cs->gpio_dev) {
-			SYS_LOG_ERR("Cannot find %s!\n",
-				    CS_CTRL_GPIO_DRV_NAME);
-			return -1;
-		}
+	cs->gpio_dev = device_get_binding(CS_CTRL_GPIO_DRV_NAME);
+	if (!cs->gpio_dev) {
```

Comment:
This function no longer takes the `cs` pointer as a parameter. Thus, these two lines should be updated to:

```
	spi_cs.gpio_dev = device_get_binding(CS_CTRL_GPIO_DRV_NAME);
	if (!spi_cs.gpio_dev) {
```